lib/Crypt/Bcrypt.pm

```diff
@@ -18,6 +18,7 @@ sub bcrypt {
 	croak "Unknown subtype $subtype" if $subtype !~ /^2[abxy]$/;
 	croak "Invalid cost factor $cost" if $cost < 4 || $cost > 31;
 	croak "Salt must be 16 bytes" if length $salt != 16;
+    croak "Password must be less than 72 bytes" if length $password > 72;
```

Comment:
The whitespace is wrong, the module uses tabs not spaces ;-)

Reply:
fixed - and s/that/than/